.github/workflows/build.yml

```diff
@@ -7,6 +7,8 @@ on:
     types:
       - created
   workflow_dispatch:
+  paths-ignore:
+    - '**.md'
```

Comment:
Doxygen uses README.md as main page, won't this prevent updating deployed docs when only README.md is changed?

Reply:
I would not know (requires testing). For the moment, lets call it out of scope of this PR. (reverted).